CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -86,6 +86,11 @@ public override void LoadContentData()
         catch (Exception e)
         {
             Log.Logger()?.ReportError(Name, ShortId, "Error retrieving data.", e);
+            var content = new JsonObject

```

Comment:
Consider promoting error message object creation to a method in the parent class (reusability + maintenance)